sotodlib/site_pipeline/make_atomic_filterbin_map.py

```diff
@@ -0,0 +1,521 @@
+    parser.add_argument("--nside",
+                        help='Nside for HEALPIX pixels')
+    parser.add_argument("--query",
+                        help='Query, can be a file (list of obs_id)\
+                        or selection string (will select only CMB scans\
+                        by default)')
+    parser.add_argument("--odir",
+                        help='Output directory')
+    parser.add_argument('--update_delay', type=int,
+                        help="Number of days (unit is days) in the past\
+                        to start observation list")
+    parser.add_argument("--nproc", type=int,
+                        help='Number of procs in\
+                        the multiprocessing pool')
+    parser.add_argument("--comps", type=str,
+                        help="Components to map (TQU by default)")
+    parser.add_argument("--singlestream", action="store_true",
+                        help="Map without demodulation (e.g. with\
+                        a static HWP)")
+    parser.add_argument("--only_hits", action="store_true",
+                        help='Only create a hits map')
+
+    # detector position splits (fixed in time)
+    parser.add_argument("--det_in_out", action="store_true")
+    parser.add_argument("--det_left_right", action="store_true")
+    parser.add_argument("--det_upper_lower", action="store_true")
+
+    # time samples splits
+    parser.add_argument("--scan_left_right", action="store_true")
+
+    parser.add_argument("--ntod", type=int, )
+    parser.add_argument("--tods", type=str, )
+    parser.add_argument("--nset", type=int, )
+    parser.add_argument("--wafer", type=str,
+                        help="Detector set to map with")
+    parser.add_argument("--freq", type=str,
+                        help="Frequency band to map with")
+    parser.add_argument("--dec_ref", type=float,
+                        help="Decl. at which we will calculate the\
+                        reference R.A.")
+    parser.add_argument("--center_at", type=str)
+    parser.add_argument("--max_dets", type=int, )
+    parser.add_argument("--fixed_ftime", type=int, )
+    parser.add_argument("--min_dur", type=int, )
+    parser.add_argument("--site", type=str, )
+    parser.add_argument("--verbose", action="count", )
+    parser.add_argument("--quiet", action="count", )
+    parser.add_argument("--window", type=float, )
+    parser.add_argument("--dtype_tod", type=str)
+    parser.add_argument("--dtype_map", type=str)
+    parser.add_argument("--atomic_db", type=str,
+                        help='name of the atomic map database, will be\
+                        saved where this script is being run')
+    parser.add_argument("--hk_data_path",
+                        help='Path to housekeeping data')
+    return parser
+
+
+def _get_config(config_file):
+    return yaml.safe_load(open(config_file, 'r'))
+
+
+def get_ra_ref(obs, site='so_sat3', dec_ref=-40):
```

Comment:
Thanks for soft-coding dec_ref.  The next step is to answer the question "what should dec_ref be".  Which we probably should have done first.

Is this RA min/max parameter used, later?  If not, let's just drop this function, and not compute the parameter.  Because it is ill-defined (at worst) and not very helpful (at best) in cases where the observation doesn't actually cross dec=-40.